src/makers/nebra/antennas.ts

```diff
@@ -26,15 +26,21 @@ const NEBRA_EU_5 = {
 } as MakerAntenna
 
 const NEBRA_EU_8 = {
-  name: '(EU 3dBi) Nebra Glass Fiber LoRa Antenna',
+  name: '(EU 8dBi) Nebra Glass Fiber LoRa Antenna',
   gain: 8,
 } as MakerAntenna
 
+const NEBRA_STOCK = {
```

Comment:
I like this pattern of not including region in the antenna description. We should apply this to all of the existing antenna definitions above too.

Reply:
Actually, there is logic to differentiate based on region: https://github.com/NebraLtd/maker-starter-app/pull/57/files#diff-5d12410e8d5096370f5ba45b0fc7ce96de653061472aa7ba0d808a8d034ea5baR27

We should:

- Add NEBRA_STOCK_US and NEBRA_STOCK_EU
- Update all hotspot structs to look like:

```
const NEBRA_STOCK_US = {
  name: 'Nebra Stock Antenna',
  gain: 3
}
const NEBRA_STOCK_EU = {
  name: 'Nebra Stock Antenna',
  gain: 3
}
...
 antenna: {
    us: ANTENNAS.NEBRA_STOCK_US,
    eu: ANTENNAS.NEBRA_STOCK_EU,
    default: ANTENNAS.NEBRA_STOCK_US,
  },
```